content/patterns/multicloud-gitops-qat/mcg-qat-getting-started.adoc

```diff
@@ -0,0 +1,305 @@
+image::multicloud-gitops-qat/qat-installed-operators.png[Multicloud GitOps Hub with qat installed operators,role="related thumb right"]
+
+
+== Deploying the cluster by using the Validated Patterns Operator
+
+To install the *Validated Patterns Operator*:
+
+. Log in to the *Openshift Container Platform* web console and select *Operators > OperatorHub*.
+
+. Search for *Validated Patterns Operator*, open it and click *Install*.
++
+.Instal Validated Patterns Operator
+image::multicloud-gitops-qat/qat-validated-pattern-operator.png[Install Validated Patterns Operator,scale=50]
++
+. Choose default settings for the installation mode, namespaces and update strategy and confirm it by clicking *Install*.
+
+. Select *Operators > Installed Operators*.
+
+. Ensure that *Validated Patterns Operator* is listed in the `openshift-operators` project with a status `Succeeded`.
+
+After succeeded installation open *Validated Patterns Operator*, go to *Pattern* tab and click *Create Pattern*.
+
+. Fill the *Name* of the pattern `multicloud-gitops-qat` and *Cluster Group Name* `hub` (from _values-global.yaml_ file).
+
+. Under *Git Config > Target Repo* copy the link to your fork and under *Git Config > Target Revision* write the name of your branch (Figure 3).
+
+. Click *Create* to create the pattern.
++
+.Create Pattern Form
+image::multicloud-gitops-qat/qat-create-pattern.png[Create pattern Multicloud GitOps with qat]
++
+
+
+Verify that the rest of Operators have been installed:
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *multicluster engine for Kubernetes* 
+* *Node Feature Discovery Operator* 
+* *Red Hat Openshift GitOps*
+* *Sail Operator*
+* *Intel Device Plugin Operator*
+
+Generate client and server certificates. User can refer to Istio official guide: https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#generate-client-and-server-certificates-and-keys
+
+After generating them add them as secrets to *Vault* manually:
+
+. Go to Vault service route. URL can be found:
+
+.. by running command:
++
+[source,terminal]
+----
+oc -n vault get route vault -ojsonpath='{.spec.host}'
+----
++
+.. in *Openshift Container Platform* web console under *Networking > Routes* for `vault` project.
+
+. Log into the Vault using root token. Root token can be found by executing command:
++
+[source,terminal]
+----
+oc -n imperative get secrets vaultkeys -ojsonpath='{.data.vault_data_json}' | base64 -d
+----
++
+
+. After login go to `secret` catalog and clik *Create secret* and fill all the fields manually (Figure 2):
+
+.. *Path for this secrets* should be look like this: `global/certs/httpbin.example.com.crt`, `global/certs/example.com.crt`, `global/certs/httpbin.example.com.key`.
```

Comment:
I don't get it even though there's a screenshot... :(
If you should put `global/certs/httpbin.example.com.crt, global/certs/example.com.crt, global/certs/httpbin.example.com.key` string into this field, please put all in one `quote`.
If not, maybe let's go with something like:
*Path for this secrets* should be filled with one of the following strings: `a`, `b`, or `c`.

Reply:
+ maybe we should change the image? It seams like it does not fit the manual.

Reply:
Yes, I agree, it's hard to understand. I didn't change image, but rephrased it, so I hope it looks much clearer.